### PR TITLE
Tokens init

### PR DESCRIPTION
**Pull request by deplanty**
Implement the tokens:

- Upload token images ;
- Set a class to load the tokens (values and images ;
- Get the tokens needed from a bet ;
- Update ConfigPaths to add tokens config ;
- Initialize the tokens after the pygame initialization (in main...)
- Add the tokens in the game_view_config
- Put an example in the game view

### CODE REVIEW

File: src/common/config.py [hunk 1]
```diff
@@ -36,4 +36,19 @@ def folder(cls, key):
...
+        Return tge path to a file given by its name
+        
+        :param str key: file name in path.cfg.json
+        :return: file path
+        :rtype: str
+        """
+
+        folder, *path = cls.path["files"][key]
+        folder = cls.folder(folder)
+        return os.path.join(folder, *path)
```
Comment (Hboni) [inline src/common/config.py]: No new line at the end.

File: src/common/func_pictures.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+from src.common.config import ConfigPath
 
 
 def load_image(file):
     """
     Loads an image, prepares it for play
     """
 
-    file = os.path.join(DIR_PICTURES, file)
+    file = os.path.join(ConfigPath.folder("pictures"), file)
```
Comment (Hboni) [inline src/common/func_pictures.py:13]: file is a "python name", use of another variable name !?

File: src/tokens/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from .tokens import Tokens
```
Comment (Hboni) [inline src/tokens/__init__.py]: No new line at the end.

File: src/tokens/tokens.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import src.common.func_pictures as fct
+from src.common.config import ConfigPath
+
+
+class Tokens:
+    data = dict()
+    
+    @classmethod
+    def initialize(cls, file):
+        with open(file) as f:
```
Comment (Hboni) [inline src/tokens/tokens.py:13]: idem file "python name"

File: src/tokens/tokens.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            cls.data = json.load(f)
+        
+        # Convert key values as integer
+        # And load images
+        for k in list(cls.data.keys()):
```
Comment (Hboni) [inline src/tokens/tokens.py:18]: Don't need of `list()` ??

File: src/tokens/tokens.py [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        :param value: researched value
+        :type value: int
+        :return: dictionnary {value: quantity}
+        :rtype: dict
+        """
+
+        values = sorted(cls.data.keys(), reverse=True)
+
+        dict_tokens = dict()
+        for i in range(len(values)):
```
Comment (Hboni) [inline src/tokens/tokens.py]: `for v in values:` ?? if you don't use i ?

File: src/tokens/tokens.py [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            v = values[i]
+            # If too much
```
Comment (ChretienCml) [inline src/tokens/tokens.py]: If enough ?  😸

**Review by Hboni — CHANGES_REQUESTED** [review]
Pretty good beginning for token use :+1:

**Review by ChretienCml — CHANGES_REQUESTED** [review]

File: src/tokens/tokens.py (hunk 6 above)
Comment (ChretienCml) [reply src/tokens/tokens.py]: `for value in values:`  ?

File: src/common/func_pictures.py (hunk 2 above)
Comment (deplanty) [reply src/common/func_pictures.py:13]: It was a keyword in 2.x versions

File: src/tokens/tokens.py (hunk 5 above)
Comment (deplanty) [reply src/tokens/tokens.py:18]: `data` is modified as I run over it and `data.keys()` returns an iterator.
So I have to convert it as a list.

**Review by Hboni — APPROVED** [review]

**Review by ChretienCml — APPROVED** [review]

**Pull request merged**